src/core/edgeNetwork/injectGetLocationHint.js

```diff
@@ -12,9 +12,20 @@ governing permissions and limitations under the License.
 
 import { getNamespacedCookieName } from "../../utils";
 import { CLUSTER } from "../../constants/cookieNameKey";
+import { MBOX_EDGE_CLUSTER } from "../../constants/legacyCookies";
 
 export default ({ orgId, cookieJar }) => {
   const clusterCookieName = getNamespacedCookieName(orgId, CLUSTER);
+  const fromClusterCookie = () => cookieJar.get(clusterCookieName);
+  const fromTarget = () => {
+    const mboxEdgeCluster = cookieJar.get(MBOX_EDGE_CLUSTER);
+    if (mboxEdgeCluster) {
+      return `t${mboxEdgeCluster}`;
```

Comment:
What's the `t` for?

Reply:
t is for target.

Reply:
@dcottingham we'll have to remove the `t` part from `mboxEdgeCluster` cookie.